portfolio-project-matching-app/components/ApplicationCard.js

```diff
@@ -71,28 +71,28 @@ const ApplicationCard = ({ appId, isOutgoing }) => {
                                     </div>
                             }
                         </div>
-                        <div className='pt-2'>
+                        <div data-testid='responseDiv' className='pt-2'>
```

Comment:
added a couple `data-testid` fields per documentation. When it is difficult to find a matcher/selector, this can be helpful.